helix-term/src/ui/editor.rs

```diff
@@ -779,7 +635,8 @@ impl EditorView {
                     );
                 }
                 text.clear();
```

Comment:
As mentioned above `text` is a mutable closure capture and therefore if we didn't clear `text` here we would accumelate all gutters (so instead of `1`, `2`, `3` the line gutters would be `1`, `12`, `123` for example)